EECS3311-PD1/src/main/java/Fetchers/FetchersCLI.java

```diff
@@ -0,0 +1,104 @@
+        System.out.println("Forest Area Fetcher\t\t\t\t\t\t\t5");
+        System.out.println("GDP Fetcher\t\t\t\t\t\t\t\t\t6");
+        System.out.println("Health Expenditure Fetcher\t\t\t\t\t7");
+        System.out.println("Hospital Beds Fetcher\t\t\t\t\t\t8");
+        System.out.println("Mortality Rate Fetcher\t\t\t\t\t\t9");
+        System.out.println("Population Fetcher\t\t\t\t\t\t\t10");
+        System.out.println("ProblemsIAHC Fetcher\t\t\t\t\t\t11");
+        fetcherValue = scanner.nextInt();
+
+        if(fetcherValue == 1){
+            AirPollutionFetcher airPollution = new AirPollutionFetcher(startYear, endYear, country);
+            dataToConsole(airPollution.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 2){
+            CO2Fetcher c02 = new CO2Fetcher(startYear, endYear, country);
+            dataToConsole(c02.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 3){
+            EducationExpenditureFetcher educationExpenditure = new EducationExpenditureFetcher(startYear, endYear, country);
+            dataToConsole(educationExpenditure.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 4){
+            EnergyUseFetcher energyUseFetcher = new EnergyUseFetcher(startYear, endYear, country);
+            dataToConsole(energyUseFetcher.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 5){
+            ForestAreaFetcher forestAreaFetcher = new ForestAreaFetcher(startYear, endYear, country);
+            dataToConsole(forestAreaFetcher.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 6){
+            GDPFetcher gdp = new GDPFetcher(startYear, endYear, country);
+            dataToConsole(gdp.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 7){
+            HealthExpenditureFetcher healthExpenditure = new HealthExpenditureFetcher(startYear, endYear, country);
+            dataToConsole(healthExpenditure.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 8){
+            HospitalBedsFetcher hospitalBeds = new HospitalBedsFetcher(startYear, endYear, country);
+            dataToConsole(hospitalBeds.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 9){
+            MortalityRateFetcher mortalityRate = new MortalityRateFetcher(startYear, endYear, country);
+            dataToConsole(mortalityRate.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 10){
+            PopulationFetcher population = new PopulationFetcher(startYear, endYear, country);
+            dataToConsole(population.getData(), startYear, endYear);
+        }
+        else if(fetcherValue == 11){
+            ProblemsIAHCFetcher problemsIAHC = new ProblemsIAHCFetcher(startYear, endYear, country);
+            dataToConsole(problemsIAHC.getData(), startYear, endYear);
+        }
+    }
```

Comment:
This works, but it would be great if you could use reflection API to *map* classes to numbers that come from the input

https://stackoverflow.com/questions/50839339/create-object-from-class-stored-in-variable